src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpSyntaxFormattingOptions.cs

```diff
@@ -59,6 +59,15 @@ internal sealed record class CSharpSyntaxFormattingOptions : SyntaxFormattingOpt
     [DataMember] public IndentationPlacement Indentation { get; init; } = IndentationDefault;
     [DataMember] public bool WrappingKeepStatementsOnSingleLine { get; init; } = true;
     [DataMember] public bool WrappingPreserveSingleLine { get; init; } = true;
+    [DataMember] public bool WrapCallChains { get; init; } = false;
+    [DataMember] public bool IndentWrappedCallChains { get; init; } = false;
+
+    [DataMember] public ParameterWrappingOptionsInternal ParameterWrapping { get; init; } = ParameterWrappingOptionsInternal.DoNotWrap;
+    [DataMember] public ParameterFirstPlacementOptionsInternal ParameterFirstPlacement { get; init; } = ParameterFirstPlacementOptionsInternal.SameLine;
+    [DataMember] public ParameterAlignmentOptionsInternal ParameterAlignment { get; init; } = ParameterAlignmentOptionsInternal.AlignWithFirst;
+
+    [DataMember] public BinaryExpressionWrappingOptionsInternal BinaryExpressionWrapping { get; init; } = BinaryExpressionWrappingOptionsInternal.DoNotWrap;
```

Comment:
note: as all the wrapping fixers are not lang specific (they work for C# and VB) the options should not be specific to C#.